src/components/form-helpers/form-helpers.js

```diff
@@ -141,14 +141,21 @@ export const FormErrors = connect(
    * @param { Object } props
    * @param { import('formik').FormikProps<Values> } props.formik
    */
-  ({ formik: { errors } }) =>
-    errors && errors['_general'] ? (
-      <div className="form-message dp-error bounceIn">
+  ({ formik: { errors } }) => {
+    const formError =
+      errors && errors['_general']
```

Comment:
maybe can ask only one time for 'errors'
`!errors || (errors['_general'] && errors['_generalWarning'])
? null
 : errors['_general'] ?   
 { message: errors['_general'], className: 'dp-error' } 
 : { message: errors['_generalWarning'], className: 'dp-warning-message' };
`
have sense?

Reply:
Mmm, it could be, for my eyes current format is more clear, but it is subjective.

Here you have both options side by side to compare:

```js
const formError =
  errors && errors['_general']
    ? { message: errors['_general'], className: 'dp-error' }
    : errors && errors['_generalWarning']
    ? { message: errors['_generalWarning'], className: 'dp-warning-message' }
    : null;
```

```js
const formError =
  !errors || !errors['_general'] || !errors['_generalWarning']
    ? null
    : errors['_general']
    ? { message: errors['_general'], className: 'dp-error' }
    : { message: errors['_generalWarning'], className: 'dp-warning-message' };
```

Reply:
thanks very much!! how can add format??

Reply:
![image](https://user-images.githubusercontent.com/1157864/59532184-64591180-8ebe-11e9-8821-6df59822f8f5.png)